salt/modules/purefb.py

```diff
@@ -0,0 +1,502 @@
+# -*- coding: utf-8 -*-
+
+##
+# Copyright 2018 Pure Storage Inc
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''
+
+Management of Pure Storage FlashBlade
+
+Installation Prerequisites
+--------------------------
+- You will need the ``purity_fb`` python package in your python installation
+  path that is running salt.
+
+  .. code-block:: bash
+
+      pip install purity_fb
+
+- Configure Pure Storage FlashBlade authentication. Use one of the following
+  three methods.
+
+  1) From the minion config
+
+  .. code-block:: yaml
+
+        pure_tags:
+          fb:
+            san_ip: management vip or hostname for the FlashBlade
+            api_token: A valid api token for the FlashBlade being managed
+
+  2) From environment (PUREFB_IP and PUREFB_API)
+  3) From the pillar (PUREFB_IP and PUREFB_API)
+
+:maintainer: Simon Dodsley (simon@purestorage.com)
+:maturity: new
+:requires: purestorage
+:platform: all
+
+.. versionadded:: Oxygen
```

Comment:
This should be updated to `Fluorine` :)

Reply:
I see some other references to `Oxygen` in this file as well, but those should all be `Fluorine`. If you could update those, too, that'd be great!